Lesson2/js/calcDatesDiff.js

```diff
@@ -0,0 +1,15 @@
+import { DateTime } from './luxon.js';

```

Comment:
тк ты установила luxon через npm, то лучше и использовать его оттуда
для этого надо переписать импорт
```suggestion
import { DateTime } from 'luxon';

```